pyeed/network/network.py

```diff
@@ -121,17 +144,24 @@ def _create_graph(self):
         self.network = copy.deepcopy(self._full_network)
         self.calculate_centrality()
 
-    def update_threshhold(self, threshold: float):
+    def update_threshhold(self, threshold: float, threshold_settings_hide: str = 'UNDER_THRESHOLD'):
```

Comment:
For simplicity and conciseness I would call this variable `mode` instead of `threshold_settings_hide`.

Reply:
or `threshold_mode`